src/sections/Community/Handbook/community.js

```diff
@@ -3,29 +3,185 @@ import { Container, Row, Col } from "../../../reusecore/Layout";
+              A Community Manager is a  person who has an innate drive to contribute to the community's prosperity. A community manager serves as a link between the organisation and its community, overcoming obstacles as they arise—or even before they arise!—by
+              collaborating with other departments and the community to find solutions that benefit the entire community.
+            </p>
+            <h3>Meshmates</h3>
+            <p>
+              Layer5 MeshMates are committed to helping community members be successful contributors. MeshMates aid in identifying areas of projects to engage within, working groups to join, and helping community members grow in their open-source and cloud-native knowledge.
+              By connecting one-on-one, MeshMates will share tips on how to have the best community experience possible.
+            </p>
+            <p>
+              MeshMate is a distinction that Layer5 awards select members of the community who innately align with the Layer5 culture of helping others, paying it forward, and have a commitment to sharing their knowledge of Layer5 projects with the community.
+              MeshMates are Layer5 ambassadors (not employees) and their commitment to helping others and sharing their expertise has a huge impact on the Layer5 community – don’t hesitate to reach out to them!
+            </p>
+
+            <h3>Contributors</h3>
+            <p>
+              As you contribute to the Layer5 community, do follow the <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">contributing guidelines</a> to make sure you are on the right track. For information about the community, please refer to the <Link to="/community/handbook/community/#communityGuidelines">community guidelines</Link>.
+              Always remember to reach out to a <Link to="/community/meshmates">MeshMate</Link> when in doubt or in need of a helping hand.
+            </p>
+
+            <h3>Badges</h3>
+            <p>
+            As you continue your contributions to this community, you stand a chance to receive a badge in acknowledgement of your engagement within and/or contribution to the representative project or (sub-)community. A variety of badges exist so that community members and their efforts may be affiliated with a particular project or community initiative.
+            </p>
+            <ul>
+              <li><img src={CGB} style={badgeStyle}/>Community</li>
+              <li><img src={LG} style={badgeStyle}/>Landscape</li>
+              <li><img src={LIH} style={badgeStyle}/>Image Hub</li>
+              <li><img src={MLL} style={badgeStyle}/>Meshery</li>
+              <li><img src={MOD} style={badgeStyle}/>Meshery Operator</li>
+              <li><img src={SDT} style={badgeStyle}/>Service Mesh Performance</li>
+              <li><img src={GNL} style={badgeStyle}/>GetNighthawk</li>
+              <li><img src={PL} style={badgeStyle}/>Patterns</li>
+            </ul>
+
```

Comment:
![R1](https://user-images.githubusercontent.com/85789734/133937100-233f40a3-8ed0-4d08-af7d-ff02423c273e.jpg)
We can remove these bullets before the icons.